src/pages/App.tsx

```diff
@@ -211,7 +219,16 @@ export default function App() {
             <Suspense fallback={<Loader />}>
               {isLoaded ? (
                 <Routes>
-                  <Route path="/" element={<Landing />} />
+                  <Route
+                    path="/"
+                    element={
+                      // if react-router-dom matches "/" with window.location.hash defined, it means that we're
+                      // using BrowserRouter and can safely redirect to a path route
```

Comment:
do you think it would be helpful to mention IPFS here as the reason why react-router-dom might not match "/" ?

Reply:
also, should we just use `isBrowserRouterEnabled()` in this file to be more explicit as to when BrowserRouter is in use?